vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go

```diff
@@ -2,12 +2,14 @@ package bridge
 
 import (
 	"errors"
+	"fmt"
 	"net"
 	"os/exec"
 	"strings"
 	"sync"
 
 	"github.com/Sirupsen/logrus"
+	bri "github.com/docker/libcontainer/netlink"
```

Comment:
@icecrime @LK4D4 This is a temporary thing to support RHEL6 and we are only pulling in libcontainer/netlink not the whole thing. 

As a side note, this is not in master(may not ever be) because if we are saying that we are going to drop support for RHEL6 in 1.8 I would rather not have this fallback logic in the master